t/gabeacon.t

```diff
@@ -55,604 +57,350 @@ my $unavailable_assembly = "GRCh38";
-
-# Is the organization is a hash
-is(ref($beacon_json->{organization}), 'HASH', 'Organization should be a hash');
-cmp_ok(keys(%{$beacon_json->{organization}}), '==', 8, 'Check organization has correct number of fields');
+# Organization is a hash
+is(ref($beacon_json->{response}->{organization}), 'HASH', 'Organization should be a hash');
+cmp_ok(keys(%{$beacon_json->{response}->{organization}}), '==', 8, 'Check organization has correct number of fields');
 
 # POST checks 
 my $json;
 
-# TODO Check for extra parameters
-
 # Testing for a variant that exists at a given location
+# includeResultsetResponses = default (HIT)
 my $post_data1 = '{"referenceName": "7", "start" : 86442403, "referenceBases": "T", "alternateBases": "C",' .
                   '"assemblyId" : "' . $assemblyId . '" }'; 
 
-my $allele_request = {
-  "referenceName" => "7",
-  "start" => "86442403",
-  "variantType" => undef,
-  "referenceBases" => "T",
-  "alternateBases" => "C",
-  "assemblyId" => $assemblyId,
-  "datasetIds" => undef,
-  "includeDatasetResponses" => undef };
-
-my $expected_data1 = {
-  "beaconId" => $beaconId,
-  "exists" => JSON::true,
-  "error" => undef,
-  "alleleRequest" => $allele_request,
-  "datasetAlleleResponses" => undef,
+my $expected_response_sum_1 = {
+  "numTotalResults" => undef,
+  "exists" => JSON::true
 };
 
-$json = json_POST( $q_base , $post_data1, 'POST dataset - 1 entry' );
-eq_or_diff($json, $expected_data1, "GA4GH Beacon query - variant at location");
-
-# Found variant with includeDataSetResponses HIT
-my $post_data1_ds = '{"referenceName": "7", "start" : 86442403, "referenceBases": "T", "alternateBases": "C",' .
-                  '"assemblyId" : "' . $assemblyId . '", "includeDatasetResponses" : "HIT", "datasetIds" : "hapmap_ceu,clin_assoc"}'; 
-
-$allele_request = {
-  "referenceName" => "7",
-  "start" => "86442403",
-  "variantType" => undef,
-  "referenceBases" => "T",
-  "alternateBases" => "C",
-  "assemblyId" => $assemblyId,
-  "datasetIds" => 'hapmap_ceu,clin_assoc',
-  "includeDatasetResponses" => 'HIT' };
-
-$dataset_response = {
-   "datasetId" => 'hapmap_ceu',
-   "exists" => JSON::true,
-   "error" => undef,
-   "frequency" => undef,
-   "variantCount" => 1,
-   "callCount" => undef,
-   "sampleCount" => undef,
-   "note" => undef,
-   "externalUrl" => $externalURL . "rs2299222",
-   "info" => undef
+my $expected_meta_receive_req_sum = {
+  "apiVersion" => $beacon_version,
+  "includeResultsetResponses" => "HIT",
+  "pagination" => { "limit" => 0, "skip" => 0 },
+  "requestedGranularity" => "record",
+  "requestedSchemas" => [ { "entityType" => "genomicVariant", "schema" => "" } ]
 };
 
-my $expected_data1_ds = {
+my $expected_meta_1 = {
+  "apiVersion" => $beacon_version,
   "beaconId" => $beaconId,
-  "exists" => JSON::true,
-  "error" => undef,
-  "alleleRequest" => $allele_request,
-  "datasetAlleleResponses" => [$dataset_response]
+  "createDateTime" => undef,
+  "includeResultsetResponses" => "HIT",
```

Comment:
Yes, I also found this difference in the specification. I decided to keep "includeResultsetResponses" because the other keys start in lowercase, for example: receivedRequestSummary